### DISCUSSION TITLE
Need help making my own Cryptocurrency

### DISCUSSION THREAD

**Discussion by AOL83** [community/community#142916] [Programming Help]
### Body

Can anyone tel me how to create my own cryptocurrency,  give me any useful information on where I can find boiler plate code for this task or anyone that has made there own please get in touch as your help would be more than greatly appreciated.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DhareshP** [comment]
you can build your own blockchain, modify an existing blockchain or build on the back of an existing blockchain

**Reply by AOL83** [reply]
Thank you for replying
